_episodes/04-lists.md

```diff
@@ -268,7 +268,7 @@ odds after reversing: [11, 7, 5, 3]
 While modifying in place, it is useful to remember that Python treats lists in a slightly
 counter-intuitive way.
 
-If we make a list and (attempt to) copy it then modify in place, we can cause all sorts of trouble:
+As we saw earlier, when we modified the (salsa) list item in-place, if we make a list, (attempt to) copy it and then modify this list we can cause all sorts of trouble. This also applies to modifying the list using the above functions:
```

Comment:
@AnnAnnFryingPan , after discussing with the other maintainers, we would like to incorporate your suggestion to refer back to the earlier time in the episode where modifying in-place is covered. Before we do so, would you mind changing `(salsa)` to `` `salsa` `` (in back ticks instead of parentheses) , and add a comma back in before `we can cause all sorts of trouble`? Thank you for your contribution!

Reply:
@Idko,  Thanks, I have done the requested updates.